tests/test_lattice.py

```diff
@@ -107,7 +141,7 @@ def test_input_validation_mismatched_lengths(self):
         # the specified exception is raised within the 'with' block.
         with pytest.raises(
             ValueError,
-            match="Identifiers and coordinates lists must have the same length.",
+            match="The number of identifiers must match the number of coordinates.",
```

Comment:
what is the point to change the text, which can only increase the diff. please keep the diff as minimal as possible